### DISCUSSION TITLE
Bluetooth crashing?

### DISCUSSION THREAD

**Discussion by ishouldbewritingcode** [PaulWoitaschek/Voice#1714] [Q&A] [answered]
With the latest version 7.0, every time I start playing a book through my Bluetooth device,  Bluetooth crashes on my phone.  Other apps seem to work fine with bluetooth. Galaxy note 9. I'll downgrade Voice to 6.5 to see if that works. 

6.5 is working fine.

Is anyone else having this issue?

**Comment by PaulWoitaschek** [comment, marked as answer]
Did you try 7.0.3?

**Reply by PaulWoitaschek** [reply]
https://github.com/PaulWoitaschek/Voice/releases/download/7.0.3/app-release.apk

**Reply by ishouldbewritingcode** [reply]
Thank you.  Seems to be working. 

Thank you for all your hard work on this program.
